### ISSUE TITLE
Translations

### ISSUE THREAD

**Issue by Firebottle** [Firebottle/Firebot#545]
Implement a system for translations of all the text in the application. 

You don't need to personally go through the entire app yourself, just implement an easy to use system and we'll catch everything as we rewrite. Reason being, a lot of the app will be rewritten before v5 is released and we'd just be overwriting your work.

**Comment by ebiggz** [comment]
Did some research and it looks like http://zanata.org/ is a good place to host our strings, once we have them, so the community can help translate.

**Comment by kateract** [comment]
Submitted the PR for this using [angular-translate](https://angular-translate.github.io/)

**Comment by Firebottle** [comment]
Does it automatically translate based on computer settings or do we need to put a toggle in somewhere (if its not already there) to switch languages?

**Comment by kateract** [comment]
I don't think so, we could provide a toggle and default it to the language in the environment if electron provides that functionality.

**Comment by Firebottle** [comment]
This might be a good entry level issue.

Basically, there is a translation file that contains name / value pairs. We need to go through the application and make sure all of the text is pulling from the translation file.

Once that is done, we can post the translation file up on different sites for people to help with translations to other languages. We then allow people to select these different translation files during the setup process.

TODO as of this comment:
- Get all of the text in the app into the translator file.
